agentdock-core/src/evaluation/evaluators/llm/judge.ts

```
@@ -0,0 +1,158 @@
+    return prompt;
+  }
+
+  /**
+   * Normalizes the raw score from the LLM based on the criterion's scale.
+   */
+  private normalizeScore(rawValue: any, scale: EvaluationScale): { score?: EvaluationResult['score'], error?: string } {
+    switch (scale) {
+      case 'binary':
+      case 'pass/fail':
+        if (typeof rawValue === 'boolean') return { score: rawValue };
+        const lowerVal = String(rawValue).toLowerCase();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)